docs/contribute/content-kitchen-sink.md

```diff
@@ -0,0 +1,245 @@
+This **bold text** and this is _italic text_.
+```
+This **bold text** and this is _italic text_.
+
+```markdown
+A link: Please visit the [Make WordPress Accessible Team Handbook](https://make.wordpress.org/accessibility/handbook/) for more info about contributor days.
+```
+A link: Please visit the [Make WordPress Accessible Team Handbook](https://make.wordpress.org/accessibility/handbook/) for more info about contributor days.
+
+```markdown
+An image: ![Alt text](url)
+for example: ![Library](/assets/images/small-image.jpg)
+```
+![Library](/assets/images/small-image.jpg)
+
+
+{: .info } 
+For more text options with Markdown read: [Basic writing and formatting syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) on GitHub.
+
+## Videos
+This site uses the accessible video player [Able Player](https://ableplayer.github.io/ableplayer/).
+
+Add to the variables at the top of the page:
+```
+has_video: true
+```
+This loads the CSS and Javascript for the Able videoplayer.
+
+Add the video itself in the page with:
+
+```markdown
+<video data-able-player data-youtube-nocookie="true" data-youtube-id="1QjSYMd_pVk" data-heading-level="0"></video>
+```
+<video data-able-player data-youtube-nocookie="true" data-youtube-id="1QjSYMd_pVk" data-heading-level="0"></video>
+
+## Code examples in the text
+
+A line of code starts and ends with a &#96; on the same line.  
+`<img src="alice.jpg" alt="White rabbit" />`
+
+A block of code starts and ends with &#96;&#96;&#96;.
+
+If you want to highlight the text for a programming language, add the language after the 3 quotes.   
+For example: &#96;&#96;&#96;html.
+
+```html
+<!-- html -->
+<img src="alice.jpg" alt="White rabbit" />
+```
+
+```javascript
+// javascript
+var foo = function lang(l) {
+  dateformat.i18n = require('./lang/' + l)
+  return true;
+}
+```
+
+```css
+/* css  */
+button {
+  cursor: pointer;
+}
+```
+
+```php
+// php
+function themename_custom_excerpt_length( $length ) { 
+    return 40; //number of words in excerpt.
+} 
+add_filter('excerpt_length','themename_custom_excerpt_length', 99 ); 
+```
+
+## Langauge switch in the content text
+
+Only use the lang attribute for 4 words and more.
+
+For a part of sentence write, use a span in your Markdown text:
+```markdown
+This text is in English and <span lang="nl">deze tekst is in het Nederlands</span> and now we continue in English.
+```
+
+In link texts
+```markdown
+[<span lang="de">Forderungspapier des Deutschen Behindertenrats zur Umsetzung des European Accessibility Acts in Deutschland</span>](https://www.deutscher-behindertenrat.de/ID255536).
+```
+
+For a larger piece of content for can add to a `<p>`, `<div>`or even a landmark, depending on what you need, for example:
+```markdown
+<main lang="de">
+[...]
+</main>
+```
+
+
+## Shortcodes
+
+ 
+```markdown
+{: .info }
+This is an informational message and uses the “info” short code.
```

Comment:
No matter what we eventually call these, it should be consistent: either "shortcode" or "short code".

Reply:
(Numerous occurrences below.)